aip/general/0121.md

```diff
@@ -53,6 +53,19 @@ database schema. In fact, having an API that is identical to the underlying
 database schema is actually an anti-pattern, as it tightly couples the surface
 to the underlying system.
 
+### Resource requirements
+
+For a valid resource-oriented API, the following must be satisfied:
+
+- The resource must have a unique [resource name][] by which it can be
+  identified and referenced.
+- The resource exposes only standard methods ([Get][], [List][], [Create][],
```

Comment:
Where these bullets correspond to sections below, it may be useful to link to those sections.

On this specific bullet, is it intended to require or preclude anything? For example, does it preclude third-party data-plane protocols?

Reply:
I am also a little confused by the intention of this point.

Reply:
It was just trying to aggregate the underlying sections, but it is applying a restriction.

Custom methods as defined by AIP 136 to provide some restrictions around the formatting of the path, methods that should be used, etc. 

I guess conversely it's not *that* restrictive anyway, since custom methods are a bunch of shoulds today. 

If you don't think it's meaningful to call out the categories of methods you're limited to, I can remove the line.

Reply:
removed.